config/manager/kustomization.yaml

```diff
@@ -16,5 +16,5 @@ apiVersion: kustomize.config.k8s.io/v1beta1
 kind: Kustomization
 images:
 - name: controller
-  newName: splunk/splunk-operator
-  newTag: 1.1.0
\ No newline at end of file
+  newName: controller
```

Comment:
are you sure this is correct? CC: @vivekr-splunk ?

Reply:
I think I pushed local change, so I reverted that to what we had originally.

Reply:
no changes to config directory is required. unless we are doing changes to CRD.